tests/scripts/functional_tests.sh

```diff
@@ -32,6 +32,7 @@ SCRIPTS_DIR="${CURRENT_DIR}"
 . ${SCRIPTS_DIR}/da/kruize_layer_id_tests.sh
 . ${SCRIPTS_DIR}/em/em_standalone_tests.sh
 . ${SCRIPTS_DIR}/remote_monitoring_tests/remote_monitoring_tests.sh
+. ${SCRIPTS_DIR}/local_monitoring_tests/local_monitoring_tests.sh
```

Comment:
Can you please check if we need updates to test_autotune.sh and update it as well.

Reply:
Update relevant test docs as well